### PR TITLE
🌐 Add French translation for `deployment/deta.md`

### PR DESCRIPTION
**Pull request by rjNemo**
🤗 

Here is the PR to translate the deployment on deta.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews

### CODE REVIEW

File: docs/fr/docs/deployment/deta.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+### Dépendances
+
+Maintenant, dans le même répertoire, créez un fichier `requirements.txt` avec :
+
+```text
+fastapi
+```
+
+!!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/deta.md:43]:
```suggestion
!!! tip "Astuce"
```

File: docs/fr/docs/deployment/deta.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+Available Commands:
+  auth        Change auth settings for a deta micro
+
+...
+```
+
+</div>
+
+!!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/deta.md:111]:
```suggestion
!!! tip "Astuce"
```

File: docs/fr/docs/deployment/deta.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+{
+        "name": "fastapideta",
+        "runtime": "python3.7",
+        "endpoint": "https://qltnci.deta.dev",
+        "visor": "enabled",
+        "http_auth": "enabled"
+}
+```
+
+!!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/deta.md:175]:
```suggestion
!!! tip "Astuce"
```

File: docs/fr/docs/deployment/deta.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+
+Allez maintenant sur <a href="https://web.deta.sh/" class="external-link" target="_blank">https://web.deta.sh</a>.
+
+Vous verrez qu'il y a une section à gauche appelée <abbr title="ça vient de Micro(server)">"Micros"</abbr> avec chacune de vos applications.
+
+Vous verrez un onglet avec "Details", et aussi un onglet "Visor", allez à l'onglet "Visor".
+
+Vous pouvez y consulter les requêtes récentes envoyées à votre application.
+
+Vous pouvez également les modifier et les réécouter.
```
Comment (Smlep) [inline docs/fr/docs/deployment/deta.md:236]:
```suggestion
Vous pouvez également les modifier et les rejouer.
```
or 
```suggestion
Vous pouvez également les modifier et les relancer.
```

**Review by Smlep — CHANGES_REQUESTED** [review]

**Review by Smlep — APPROVED** [review]

**Review by axel584 — APPROVED** [review]

**Comment by tiangolo** [comment]
Great, thanks @rjNemo! :cake: 

And thanks for the reviews @axel584 and @Smlep! :coffee:

**Pull request merged**